bedrock/products/templatetags/misc.py

```diff
@@ -39,24 +39,15 @@ def _vpn_get_ga_data(selected_plan):
     analytics = selected_plan.get("analytics")
 
     ga_data = (
-        "{"
-        "'id' : '%s',"
-        "'brand' : '%s',"
-        "'plan' : '%s',"
-        "'period' : '%s',"
-        "'price' : '%s',"
-        "'discount' : '%s',"
-        "'currency' : '%s'"
-        "}"
-        % (
-            id,
-            analytics.get("brand"),
-            analytics.get("plan"),
-            analytics.get("period"),
-            analytics.get("price"),
-            analytics.get("discount"),
-            analytics.get("currency"),
-        )
+        f"{{"
```

Comment:
Should this be a double?

Reply:
Yeah if you look at the original we output a `{` and `}`. For f-strings to do that you double it.

Reply:
There's tests for this also btw.